include/xsimd/types/xsimd_api.hpp

```diff
@@ -1375,6 +1390,21 @@ namespace xsimd
         return kernel::proj(z, A {});
     }
 
+    /**
+     * @ingroup batch_arithmetic
+     *
+     * Computes the approximate reciprocal of the batch \c x.
+     * The maximum relative error for this approximation is
+     * less than 1.5*2^-12.
+     * @param x scalar or batch of scalars
+     * @return the reciprocal.
+     */
+    template <class A>
+    inline batch<float, A> reciprocal(batch<float, A> const& x) noexcept
```

Comment:
no `double` version? Even if it means using the default algorithm, for genericity I think we need one.

Reply:
I've patched it by requiring SFINAE if T is not a floating point type, is that OK with you?